app/views/share_obituary_mailer/inform.html.erb

```diff
@@ -1,7 +1,7 @@
 Hello <%=@recipient_name%>,
 <br>
 <br>
-<%= @sender.first_name %> would like to share this obituary with you.<a href="/obituaries/<%=@obit_id%>" > link to obituary</a>
+<%= @sender.first_name %> would like to share this obituary with you, <a href='https://obituary-hub.herokuapp.com/obituaries/<%=@obit_id%>' > link to obituary.</a>
```

Comment:
These links should work now, I double checked with mail catcher.

Reply:
![](https://media0.giphy.com/media/MhHXeM4SpKrpC/giphy.gif?cid=5a38a5a2l04xgh2hfjm00y3vs4tbtfkxlkf0ifighf3p911k&rid=giphy.gif)